blocks/test/full-content.js

```diff
@@ -124,10 +123,7 @@ describe( 'full post content fixture', () => {
 			const parserOutputExpected = JSON.parse( parserOutputExpectedString );
 			expect(
 				parserOutputActual
-			).to.eql(
-				parserOutputExpected,
-				format( 'File \'%s.parsed.json\' does not match expected value', f )
-			);
+			).toEqual( parserOutputExpected );
```

Comment:
When these tests fail, debugging them is pretty tricky without these messages in place.  Let's preserve them (this one and the 2 below it), even if we have to do it via a `try / catch` or some other non-pretty mechanism.

Context: https://github.com/facebook/jest/issues/1965